webapp/src/generalSlices/rsuSlice.ts

```diff
@@ -1,567 +1,568 @@
+          ;(state.value.rsuOnlineStatus as RsuOnlineStatusRespMultiple)[payload.ip]['last_online'] = payload.last_online
+        }
+      })
+      .addCase(getRsuLastOnline.rejected, (state) => {
+        state.loading = false
+      })
+      .addCase(_getRsuInfo.fulfilled, (state, action) => {
+        state.value.rsuData = action.payload
+      })
+      .addCase(_getRsuOnlineStatus.fulfilled, (state, action) => {
+        state.value.rsuOnlineStatus = action.payload as RsuOnlineStatusRespMultiple
+      })
+      .addCase(_getRsuCounts.fulfilled, (state, action) => {
+        state.value.rsuCounts = action.payload.rsuCounts
+        state.value.countList = action.payload.countList
+      })
+      .addCase(_getRsuMapInfo.fulfilled, (state, action) => {
+        state.value.startDate = action.payload.startDate
+        state.value.endDate = action.payload.endDate
+        state.value.mapList = action.payload.rsuMapData
+      })
+      .addCase(getSsmSrmData.pending, (state) => {
+        state.loading = true
+      })
+      .addCase(getSsmSrmData.rejected, (state) => {
+        state.loading = false
+      })
+      .addCase(getSsmSrmData.fulfilled, (state, action) => {
+        state.value.srmSsmList = action.payload
+      })
+      .addCase(getIssScmsStatus.fulfilled, (state, action) => {
+        state.value.issScmsStatusData = action.payload ?? state.value.issScmsStatusData
+      })
+      .addCase(updateRowData.pending, (state) => {
+        state.requestOut = true
+        state.value.messageLoading = false
+      })
+      .addCase(updateRowData.fulfilled, (state, action) => {
+        if (action.payload === null) return
+        state.value.rsuCounts = action.payload.rsuCounts
+        state.value.countList = action.payload.countList
+        state.value.heatMapData.features.forEach((feat, index) => {
+          const ip = feat.properties.ipv4_address as string
+          state.value.heatMapData.features[index].properties.count =
+            ip in action.payload.rsuCounts ? action.payload.rsuCounts[ip].count : 0
+        })
+        state.value.warningMessage = action.payload.warningMessage
+        state.requestOut = false
+        state.value.messageLoading = false
+        state.value.countsMsgType = action.payload.countsMsgType
+        state.value.startDate = action.payload.startDate
+        state.value.endDate = action.payload.endDate
+      })
+      .addCase(updateRowData.rejected, (state) => {
+        state.requestOut = false
+        state.value.messageLoading = false
+      })
+      .addCase(updateGeoMsgData.pending, (state) => {
+        console.debug('updateGeoMsgData pending')
+        state.loading = true
+        state.value.addGeoMsgPoint = false
+        // Removed 1 day limitation for new mongo deployment
+        // state.value.geoMsgDateError =
+        //   new Date(state.value.geoMsgEnd).getTime() - new Date(state.value.geoMsgStart).getTime() > 86400000
```

Comment:
Are these comments still relevant?

Reply:
I am not sure... my git merge from develop seems to have screwed up my git lens. Let me check with the author on this!

Reply:
Checked with Michael, removed!